retrograde-util/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+}
+
+dependencies {
+    api(deps.libs.timber)
+
+    implementation(deps.libs.jna)
+    implementation(deps.libs.koptional)
+    implementation(deps.libs.koptionalRxJava2)
+    implementation(deps.libs.kotlinStdlib)
+    implementation(deps.libs.okHttp3)
+    implementation(deps.libs.roomRuntime)
+    implementation(deps.libs.rxJava2)
+    implementation(deps.libs.supportAppCompatV7)
+    implementation(deps.libs.supportRecyclerViewV7)
+}
+
+android { }
```

Comment:
Is this necessary?

Reply:
I kept original files structure, it can be easily removed